src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```
@@ -12,11 +16,32 @@ public ActorDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Actor add(Actor actor) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.save(actor);
+            transaction.commit();
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new DataProcessingException("Can't add the actor: " + actor, e);
+        } finally {
+            if (session != null) {
+                session.close();
+            }
+        }
+        return actor;
     }
 
     @Override
     public Optional<Actor> get(Long id) {
-        return null;
+        try (Session session = factory.openSession()) {
+            return Optional.of(session.get(Actor.class, id));
+        } catch (HibernateException | NullPointerException e) {
+            return Optional.empty();
+        }
```

should we return empty optional or is it better to throw a runtime exception?